command/upgrade/provider/deno_land.ts

```diff
@@ -6,8 +6,8 @@ export interface DenoLandProviderOptions {
 
 export class DenoLandProvider extends Provider {
   name = "deno.land";
-  private readonly repositoryUrl = "https://deno.land/x/";
-  private readonly registryUrl = "https://deno.land/x/";
+  private readonly repositoryUrl = "https://deno.land/x";
+  private readonly registryUrl = "https://deno.land/x";
```

Comment:
Was this change intended or is this a bug in deno-outdated? Because the trailing slash is required here.

Reply:
Got it, I'll use udd instead for this PR.

Reply:
I ended up still using deno-outdated until my discussion in deno is resolved: https://github.com/denoland/deno/discussions/15625 as globs can't be passed to check all the *.ts files in this repository. When this is resolved, I'll make another PR to switch to udd.

Reply:
Thx for fixing 👍